config/environments/development.rb

```diff
@@ -14,7 +14,21 @@
   config.action_controller.perform_caching = false
 
   # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
+  #config.action_mailer.raise_delivery_errors = false
+  # Mailer default settings
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.default_url_options = {
+    :host => "localhost:3000"
+  }
+  # SMTP settings for gmail
+  config.action_mailer.smtp_settings = {
+    :address              => "smtp.gmail.com",
+    :port                 => 587,
+    :user_name            => ENV['gmail_username'],
+    :password             => ENV['gmail_password'],
+    :authentication       => "plain",
+    :enable_starttls_auto => true
+  }
```

Comment:
Will emails work in production? I see you only edited the development config file.